features/step_definitions/user_steps.rb

```diff
@@ -35,14 +30,9 @@
 
 Then(/^I should see that I am logged in as a "([^"]*)"$/) do |role|
   visit admin_user_path(@user)
-  click_link "#user_settings"
```

Comment:
I haven't tried running this but I would have expected this line to still be necessary since it's outside the `if` clause.

Reply:
Hi David as part of new implementation we do have user setting link anymore.We will check it again though.

Reply:
@davidtrussler The line above is already visiting the user settings page, I believe? So clicking the link takes them to the page they're already on.

If the desire is to test the link then maybe we should be visiting another page and following the link from there, otherwise I don't think we need the link…thoughts?

Reply:
@farahTW / @mtaylorgds Yes, that's fine. Probably the link shouldn't have been there still but anyway LGTM.